internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,70 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+)
+
+var (
+	ErrInvalidArgumentsCount = errors.New("invalid arguments count")
+	ErrInvalidFormat         = errors.New("invalid format")
+	ErrZeroOrNegativeValue   = errors.New("zero or negative value")
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return fmt.Errorf("%w: %s", ErrInvalidArgumentsCount, datastring)
+	}
+
+	ds.Steps, err = strconv.Atoi(data[0])
+	if err != nil {
+		return fmt.Errorf("%w: %s", ErrInvalidFormat, data[0])
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```